EventFilter/CTPPSRawToDigi/src/RPixErrorChecker.cc

```diff
@@ -12,41 +12,53 @@
 using namespace std;
 using namespace edm;
 
-namespace {
-  constexpr int CRC_bits = 1;
-  constexpr int ROC_bits  = 5;
-  constexpr int DCOL_bits = 5;
-  constexpr int PXID_bits = 8;
-  constexpr int ADC_bits  = 8;
-  constexpr int OMIT_ERR_bits = 1;
+   constexpr int RPixErrorChecker::CRC_bits ;
```

Comment:
I miss why these contexpr must be defined twice, in the header and here.
Let see what the compiler says...

Reply:
If I remove the "duplication" the compiler (linker actually) complains. This "duplication" appears in other CMSSW areas.

Reply:
the complaints should show up only if there is an attempt to use address of some of these values (ODR use).
A somewhat incomplete check shows only dummyDetId that may be affected.

Reply:
Now I understand a bit better, thanks. I removed all the duplications from .cc. I just left dummyDetId.